src/Provider/Provider.Web/RouteModel/VacancyRouteModel.cs

```diff
@@ -5,16 +5,10 @@ namespace Esfa.Recruit.Provider.Web.RouteModel
 {
     public class VacancyRouteModel
     {
-        private string _ukprn;
-
         [FromRoute]
-        public string Ukprn
-        {
-            get => _ukprn;
-            set => _ukprn = value.ToUpper();
-        }
+        public long Ukprn { get; set; }
```

Comment:
Should this be string so we don't get model binder type issues?

Reply:
Thinking about this, we should be reading it from the user, but I am not sure if that is possible.